docs/src/main/sphinx/connector/hive.md

```diff
@@ -30,6 +30,9 @@ implementation of the Hive metastore, such as
 Many [distributed storage systems](hive-file-system-configuration) can be
 queried with the Hive connector.
 
+You must select and configure a supported [file
```

Comment:
Isn't this kind of duplicating the prior sentence so we should choose either one